src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -232,7 +232,7 @@ private void DiscoverHubMethods()
         private static bool IsHubMethod(MethodInfo m)
         {
             // TODO: Add more checks
-            return m.IsPublic;
+            return m.IsPublic && typeof(Hub<IClientProxy>) != m.GetBaseDefinition().DeclaringType;
```

Comment:
This isn't right. You want to look for an open generic `Hub<>`, also, why do we care about this specifically? Did you look at what old signalr did?